playbooks/roles/capsule/tasks/devel_install.yml

```diff
@@ -0,0 +1,17 @@
+- name: 'start rails server'
+  command: /home/vagrant/.rvm/bin/rvm ruby-2.2.4 do rails s
+  args:
+    chdir: /home/vagrant/foreman
+  delegate_to: "{{ capsule_server }}"
+  async: 1000
+  poll: 0
+  become: yes
+  become_user: vagrant
+
+- name: 'Add group foreman'
+  group: name=foreman state=present
+  delegate_to: "{{ capsule_server }}"
+
+- name: 'Install foreman-installer-katello'
+  yum: name="foreman-installer-katello"
+  delegate_to: "{{ capsule_server }}"
```

Comment:
I took it out since you had the packaging PR up, but added it back in since I don't think we should hold this PR for that one